src/cli/parse-flags.ts

```diff
@@ -454,22 +457,35 @@ const throwNumberParsingError = (flag: string, value: string) => {
 };
 
 /**
- * A little helper to 'dereference' a flag alias, which if you squint a little
- * you can think of like a pointer to a full flag name. Thus 'c' is like a
- * pointer to 'config', so here we're doing something like `*c`. Of course, this
- * being JS, this is just a metaphor!
+ * A little helper to 'desugar' a flag alias, meaning expand it to its full
+  * name. For instance, the alias `"c"` will desugar to `"config"`.
  *
- * If no 'dereference' is found for the possible alias we just return the
- * passed string unmodified.
+ * If no expansion is found for the possible alias we just return the passed
+ * string unmodified.
  *
  * @param maybeAlias a string which _could_ be an alias to a full flag name
  * @returns the full aliased flag name, if found, or the passed string if not
  */
-const dereferenceAlias = (maybeAlias: string): string => {
-  const possibleDereference = CLI_FLAG_ALIASES[maybeAlias];
+const desugarAlias = (maybeAlias: string): string => {
```

Comment:
I renamed this because I think desugaring is a more apt analogy than dereferencing